src/test/resources/features/resolve/after_pep_object_resolve.feature

```diff
@@ -0,0 +1,95 @@
+Feature: Resolve after pep object
+
+  Scenario Outline: Resolve at toplevel
+    Given a snakemake project
+    Given I open a file "foo.smk" with text
+    """
+    pep.<property>
+    """
+    When I put the caret after pep.
+    Then reference should resolve to "<property>" in "project.py"
+
+    Examples:
+      | property                   |
+      | __init__                   |
```

Comment:
Again no need to test resolve for all class methods. E.g nobody will use `__init__` method after `pep.<>`.  Use only reasonable methods. Check peppy/snakemake documentation to figure out which methods are important API

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/424/commits/dcaea2a9a74dff8d433706ce1fff2d61620318b4